Jenkinsfile

```diff
@@ -150,6 +150,20 @@ pipeline {
                             }
                         }
 
+                        stage('RTL Tests') {
```

Comment:
this changes was deleted by mistake in previous PR of mine, so just re-implementing